File: ios/RCTUITestManager.swift
//
//  RCTUITestManager.swift
//  RNExample
//
//  Created by puti on 2018/4/24.
//  Copyright © 2018年 Facebook. All rights reserved.
//

import Foundation

@objc(RCTUITest)
class RCTUITestManager: RCTViewManager {
  override func view() -> UIView! {
    return RCTUITest()
  }
  
  
  
}

File: android/customlib/src/main/java/com/example/customlib/RCTUITestManager.java
package com.example.customlib;

import com.facebook.react.uimanager.SimpleViewManager;
import com.facebook.react.uimanager.ThemedReactContext;
import com.facebook.react.uimanager.annotations.ReactProp;

/**
 * Created by puti on 2018/1/14.
 */

public class RCTUITestManager extends SimpleViewManager<RCTUITest> {

    @Override
    public String getName() {
        return "RCTUITest";
    }

    @Override
    protected RCTUITest createViewInstance(ThemedReactContext reactContext) {
        return new RCTUITest(reactContext);
    }

    @ReactProp(name = "customText")
    public void setCustomText(RCTUITest view, String text) {
        view.setText(text);
    }

}
